src/renderer/components/ft-share-button/ft-share-button.vue

```diff
@@ -26,13 +26,27 @@
           <font-awesome-icon icon="copy" />
           {{ $t("Share.Copy Link") }}
         </ft-button>
+        <ft-button
```

Comment:
Instead of new buttons in the share popup, let's remove these and include a `ft-toggle-switch` for determining if the user wants to get the link at the current time or not.

I variable would be set within the component to false by default, something like `includeTimestamp`.  The `ft-toggle-switch` would then change the variable when toggled and your copy / open logic will be determined by that instead.  This would also remove your `copyXXXAtTime()` functions as well.

Reply:
Sure, I can do that, but could you please provide a line in the file where the `ft-toggle-switch` should go? I'd rather not guess at updating your UI - the only reason I made an arbitrary decision on where to put the new buttons was because I was mimicking the structure of the YouTube right-click menu.

Reply:
Preferably above both columns with the toggle switch centered.  You can wrap it in a `ft-flex-box` component to handle most of the logic for you.  If you're still unsure then let me know and I can push to your branch and add the toggle switch